discovery-server/src/main/asciidoc/application-config-guide.adoc

```diff
@@ -146,5 +146,10 @@ polaris:
       timeout: 20       # Timeout for rule edit sessions
     etl:
       timeout: 36000    # Timeout for generating snapshots
+  extensions:
+      lnb:
```

Comment:
@minhyun2 you should be mentioned that the menu can be expanded in the lnb area.

Reply:
@kyungtaak 
I added a comment as below.
# Specify the menu to add to the LNB.